docs/src/main/asciidoc/flyway-guide.adoc

```diff
@@ -76,28 +125,50 @@ INSERT INTO quarkus(id, name)
 VALUES (1, 'QUARKED');
 --
 
-Now you can create your `Flyway` object as you usually do:
+Now you can start your application and {project-name} will run the Flyway's migrate method according to your config:
+
+[source,java]
+--
+@ApplicationScoped
+public class MigrationService {
+    // You can Inject the object if you want to use it manually
+    @Inject
+    Flyway flyway; <1>
+
+	public void checkMigration() {
+        // This will print 1.0.0
+        System.out.println(flyway.info().current().getVersion().toString());
+    }
+}
+--
+
+<1> Inject the Flyway object if you want to use it directly
+
+== Using the Flyway object
+
+In case you are interested in using the `Flyway` object directly, you can inject it as follows:
+
+NOTE: If you enabled the `quarkus.flyway.migrate-at-start` property, by the time you use the Flyway instance,
+{project-name} will already have run the migrate operation
 
 [source,java]
 --
 @ApplicationScoped
 public class MigrationService {
+    // You can Inject the object if you want to use it manually
+    @Inject
+    Flyway flyway; <1>
 
-    @ConfigProperty(name = "datasource.url") <1>
-    String dbURL;
-    @ConfigProperty(name = "datasource.username")
-    String dbUser;
-    @ConfigProperty(name = "datasource.password")
-    String dbPassword;
-
-    public void migrate() {
-        Flyway flyway = Flyway.configure()
-                        .dataSource(dbURL, dbUser, dbPassword)
-                        .load(); <2>
+	public void checkMigration() {
```

Comment:
Sorry, it's very minor but there is an indentation issue here. Could you fix it?

Reply:
Sure, Will push in a minute. I have some free time